spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -799,3 +697,4 @@ def set_info_page(self):
                 self.info_page,
                 QUrl.fromLocalFile(self.css_path)
             )
+            self.sig_execution_state_changed.emit()
```

Comment:
Why is this necessary now?

Reply:
It was in `_finalise_restart` which was removed

Reply:
Also it makes sense that the execution state is changed when an info page is shown.